### ISSUE TITLE
Alas doesn't compile on win7

### ISSUE THREAD

**Issue by gilescope** [tarka/xcp#2]
Compiling xcp v0.4.0
error[E0433]: failed to resolve: could not find `unix` in `os`
  --> xcp-0.4.0\src\operations.rs:22:14
   |
22 | use std::os::unix::fs::symlink;
   |              ^^^^ could not find `unix` in `os`

error[E0425]: cannot find function `symlink` in this scope
   --> xcp-0.4.0\src\operations.rs:118:26
    |
118 |                 let _r = symlink(&from, &to);
    |                          ^^^^^^^ not found in this scope

error: aborting due to 2 previous errors

**Comment by tarka** [comment]
Yeah, kind-of expected. Unfortunately I've never managed to get Rust to install properly on Win10, so I'm not in a position to fix it myself.
